src/components/HeaderWithBackButton/index.js

```diff
@@ -21,7 +22,7 @@ import ROUTES from '@src/ROUTES';
 import headerWithBackButtonPropTypes from './headerWithBackButtonPropTypes';
 
 function HeaderWithBackButton({
-    iconFill = undefined,
+    iconFill = null,
```

Comment:
@WojtekBoman, what was the reason for this update? I'm working on the [TS migration](https://github.com/Expensify/App/pull/33214) of this file and want to be sure I haven't missed something.